spec.mediawiki

```diff
@@ -1146,59 +1322,105 @@ transaction. They broadcast their transactions.
 transactions, one for Bob and one for Carl. 
 Bob and Carl then follow up with their own redemptions.
 
-===Community Conduct===
+==Community Conduct==
 
 By registering, clients agree to abide by the rules described here. These rules 
 are designed to ensure that clients are acting in good faith and maintaining 
 behavior that contribute to a smooth DEX experience for humans. 
 
-====Rules of Community Conduct====
+===Rules of Community Conduct===
 
-'''Rule 1: You must act on a match when it's your turn'''
+====Rule 1: Every match must be fully settled
 
 In the event that the taker fails to start the atomic swap process within a 
 pre-defined timeout period with their initialization transaction, the maker 
 will be notified that order execution is terminated due to failure of the taker 
 to accept the order. The Maker's limit order will not go 
 back on the order book, but they may be given the option to replace the order. 
 
-In the event that the Maker fails to respond within a pre-defined timeout period 
-to the taker's initialization transaction with an initialization transaction of 
-their own, the taker will be given the option to place a new market order.
+In the event that the taker fails to respond to the maker's initialization 
+transaction, the maker will incur no violation.
+
+====Rule 2: An account's cancellation ratio must remain below the threshold
+
+The cancellation ratio is the ratio of the count of cancelled orders to the 
+count of completed orders on an account. The cancellation threshold is set
+by the DEX operator.
+An order is considered completed when all matches have fully settled. 
+An order is considered cancelled when a cancel order is matched to a standing 
+limit order, or when a connection is dropped before an order is filled.
```

Comment:
As I mentioned before, we may wish to penalize dropped connection cancels more (double?), at least if it happens repeatedly, which suggests the user is using disconnects to achieve order cancellation having their cancel order enter the epoch matching queue.  That is a trick I think we should discourage.